### PR TITLE
[RFR] Doc: how to use dev version in real world project?

### PR DESCRIPTION
**Pull request by jpetitcolas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,3 +130,25 @@ Check `sideEffect/saga.js` for a detail of the inputs and outputs
 * Add unit tests
 * Add `<ReferenceMany>` field
 * Add complex field & input types
+
+## Contributing
```
Comment (greg0ire) [inline README.md]: How about putting that in a `CONTRIBUTING.md` file? Github / Gitlab have special integration features for this.

File: README.md [hunk 2]
```diff
@@ -130,3 +130,25 @@ Check `sideEffect/saga.js` for a detail of the inputs and outputs
...
+
+### Testing Your Changes on a Real Project
+
+In order to test your changes on a real world project, you can embed a work-in-progress version of the lib in another repository, using `npm-link`:
```
Comment (greg0ire) [inline README.md]: Why a dash in `npm-link` ?

File: README.md [hunk 3]
```diff
@@ -130,3 +130,25 @@ Check `sideEffect/saga.js` for a detail of the inputs and outputs
...
+npm link # allow this repo to be linked from another project
+
+cd ../real-world-project
+npm link admin-on-rest # replace admin-on-rest module by a symbolic link to dev version
+```
+
+To use the real version again, just remove your symbolic link and reinstall dependencies:
+
+``` sh
+rm -Rf node_modules/admin-on-rest
```
Comment (greg0ire) [inline README.md]: `unlink` should be enough, right?

File: README.md (hunk 1 above)
Comment (fzaninotto) [inline README.md]: Let's keep that in the README for now, `Contributing.md` is more a guideline for pull requests. At that step, it's probable that contributors will have already tested their changes (I hope).

File: README.md (hunk 3 above)
Comment (fzaninotto) [inline README.md]: nah, let's keep `rm`

File: README.md (hunk 2 above)
Comment (fzaninotto) [inline README.md]: That's how it's described in https://docs.npmjs.com/cli/link

**Comment by fzaninotto** [comment]
Thanks!

File: README.md (hunk 2 above)
Comment (jpetitcolas) [inline README.md]: ~~That's a typo.~~

It is for reffering to official documentation (thanks @fzaninotto :p).

File: README.md (hunk 3 above)
Comment (greg0ire) [inline README.md]: Ok, but then why would you need to make it recursive? IMO this is not the command that would come to mind in association with "remove your symbolic link".

**Pull request merged**